docs/observability/platform-dashboard.mdx

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 3
+sidebar_label: Platform Dashboard
+description:
+  "The Platform Dashboard is a comprehensive reporting tool designed to provide insights into the performance, security,
+  and usage of the Data Delivery Netowork"
+keywords:
+  - hasura ddn
+  - metrics
+  - performance
+  - latency
+  - request rate
+  - error rate
+  - graphql api
+  - opentelemetry
+  - application debugging
+  - application performance
+  - api monitoring
+  - platform dashboard
+  - dashboard
+  - application management
+---
+
+import Thumbnail from "@site/src/components/Thumbnail";
+
+# Platform Dashboard
+
+Go to the `Platform Report` tab in the `Insights` panel for your project to view the new platform metrics. This
+dashboard provides a visual overview of key metrics across four main categories, offering a comprehensive assessment of
+the Data Delivery Network platform:
+
+## Security and Governance
+
+:::tip Available on
+
+Security and Governance reports are available on DDN Free plans and up.
+
+:::
+
+### Public Endpoint Alert
+
+Quickly identify any models that are publicly exposed without authentication.
+
+This feature helps prevent unauthorized access by providing clear notifications of potential vulnerabilities. By
+scanning your API endpoint for any publicly accessible models, you’ll be alerted if any part of your system is not
+secured, allowing you to take immediate action.
+
+### Security Issue Resolution
+
+A step-by-step guide to resolve detected security issues.
+
+This feature provides actionable insights on how to secure your supergraph, helping you implement necessary changes
+without guesswork. Each detected issue is paired with recommended fixes to improve the overall security of your API.
+
+<Thumbnail
+  src="/img/get-started/dashboard_security.png"
+  alt="The Security and Governance report showing that NoAuth mode makes the API publicly accessible"
+  width="1000"
+/>
```

Comment:
@julian-mayorga and @rahulagarwal13 — please note what I did here to make things clearer. You can see more details of the step-by-step we need to replicate for the other subheadings in this commit's [description](https://github.com/hasura/ddn-docs/pull/720/commits/2acee19403567ee9fc76425cfaf7c905acf0c701).